dali/test/python/test_operator_arithmetic_ops.py

```diff
@@ -241,14 +258,6 @@ def iter_setup(self):
         for i in range(len(inputs)):
             self.feed_input(self.source[i], inputs[i])
 
-    # Workaround for unary `+` operator
```

Comment:
Can we now?

Reply:
I think so, I also think it was your change :P

Reply:
Those moments are lost in time....